[doc] Add install.rst for installation compatability troubleshooting (#980)
taichi-dev/taichi: docs/hello.rst

Hello, world!
=============

We introduce the Taichi programming language through a very basic `fractal` example.

> Running the Taichi code below (``python3 fractal.py``) will give you a image of

Reviewer: ```suggestion
Running the Taichi code below (``python3 fractal.py``) will give you an animation of
```
